emitter.py

```diff
@@ -29,7 +30,13 @@
 
 
 def remove_control_chars(s):
-    return control_char_re.sub('', s)
+    if isinstance(s, str):
+        sanitized = control_char_re.sub('', s)
+    elif isinstance(s, unicode):
+        sanitized = ''.join(['' if unicodedata.category(c) == 'Cc' else c
```

Comment:
Is `Cc` the only category failing ? http://www.fileformat.info/info/unicode/category/index.htm